diff --git a/dist/components/ForceSpecificCursor.js b/dist/components/ForceSpecificCursor.js
index c915f57c35f20e42dfe6aac2f407fc4b69f5e6db..7992f9fe361ae49b7aa71c381988ff1a622f7227 100644
--- a/dist/components/ForceSpecificCursor.js
+++ b/dist/components/ForceSpecificCursor.js
@@ -39,7 +39,7 @@ const react_1 = __importStar(require("react"));
 const ref = react_1.default.createRef();
 const forceSpecificCursor = (cursor) => {
     if (!ref.current) {
-        throw new Error('ForceSpecificCursor is not mounted');
+        return;
     }
     ref.current.style.cursor = cursor;
     ref.current.style.pointerEvents = 'auto';
@@ -47,7 +47,7 @@ const forceSpecificCursor = (cursor) => {
 exports.forceSpecificCursor = forceSpecificCursor;
 const stopForcingSpecificCursor = () => {
     if (!ref.current) {
-        throw new Error('ForceSpecificCursor is not mounted');
+        return;
     }
     ref.current.style.cursor = '';
     ref.current.style.pointerEvents = 'none';
diff --git a/dist/components/PlaybackRateSelector.js b/dist/components/PlaybackRateSelector.js
index 852be2658dd342ea384ee945712830e81c79d126..331d5dafa5039157719b956d8a20b329a896a733 100644
--- a/dist/components/PlaybackRateSelector.js
+++ b/dist/components/PlaybackRateSelector.js
@@ -10,7 +10,7 @@ const playbackrate_1 = require("../state/playbackrate");
 const ControlButton_1 = require("./ControlButton");
 const ComboBox_1 = require("./NewComposition/ComboBox");
 const commonPlaybackRates = [
-    -4, -2, -1, -0.5, -0.25, 0.25, 0.5, 1, 1.5, 2, 4,
+    -4, -2, -1, -0.5, -0.25, 0.25, 0.5, 1, 1.05, 1.1, 1.12, 1.15, 1.25, 1.5, 2, 4,
 ];
 const getPlaybackRateLabel = (playbackRate) => {
     return `${playbackRate}x`;
diff --git a/dist/components/draw-peaks.js b/dist/components/draw-peaks.js
index ab518e6e10a307fb8a90589703b443c5b2a5f814..117acfe777d506d449caaa3117321db101d4e640 100644
--- a/dist/components/draw-peaks.js
+++ b/dist/components/draw-peaks.js
@@ -13,6 +13,8 @@ const drawBars = (canvas, peaks, color, volume, width) => {
     ctx.clearRect(0, 0, w, height);
     if (volume === 0)
         return;
+    if (w <= 0 || height <= 0)
+        return;
     const [r, g, b, a] = (0, parse_color_1.parseColor)(color);
     const [cr, cg, cb, ca] = (0, parse_color_1.parseColor)(CLIPPING_COLOR);
     const imageData = ctx.createImageData(w, height);
